pip/pep425tags.py

```diff
@@ -127,6 +127,54 @@ def get_platform():
     return distutils.util.get_platform().replace('.', '_').replace('-', '_')
 
 
+def is_manylinux1_compatible():
+    # Only Linux, and only x86-64 / i686
+    if get_platform() not in ("linux_x86_64", "linux_i686"):
+        return False
+
+    # "wide" Unicode mode is mandatory (always true on CPython 3.3+)
+    if sys.maxunicode <= 0xFFFF:
+        return False
+
+    # Check for presence of _manylinux module
+    try:
+        import _manylinux
+        return bool(_manylinux.manylinux1_compatible)
```

Comment:
No, this changes the semantics -- the reason it's written that way in the PEP is that it's legal to have a `_manylinux` module that doesn't define a `manylinux1_compatible` attribute. (Hence the `AttributeError` check.)